core/src/main/java/io/kestra/core/storages/StorageInterface.java

```diff
@@ -30,6 +30,13 @@ public interface StorageInterface {
     @Retryable(includes = {IOException.class}, excludes = {FileNotFoundException.class})
     InputStream get(String tenantId, URI uri) throws IOException;
 
+    /**
+     * Returns all existing file paths' that start with the given prefix.
+     * @return relative paths starting from prefix
```

Comment:
The second line is misleading, does it retruns path or path relative to the preifx?
I think it should return full URI

Reply:
So the test shows that it returns relative path which is strange.
Looking at the interface of the LocalStorage I think it should returns a List<URI> instead, containing the full internal storage URI.
This if you need a list of string without the prefix it is something related to NamespaceFiles so it must be adapted inside the NamespaceFile service.
Otherwise, this method will not be reusable for something else.

Reply:
replied in the other comment